### PR TITLE
WOW Project updates

### PR DESCRIPTION
**Pull request by ashiq-r31**
Apologies for messing up the table markdown with formatting...

- Added CRUD operations as requirement in proposal process
- Updated link for Courseworks assignment
- Updated submission dates
- Updated wireframing section with Miro as the tool

### CODE REVIEW

**Review by YusufBolden — APPROVED** [review]
I will delete the reference to the Blog app as Lillie will teach the functionality of the blog app within the Todo app.

File: projects/wow/wow.md [hunk 1]
```diff
@@ -2,203 +2,207 @@
...
-- Django [Models](https://github.com/Justice-Through-Code/fall_2021/blob/main/lessons/django_todo/lesson.md)
-- Django [Models with many-to-many and one-to-many relationships](https://github.com/Justice-Through-Code/fall_2021/blob/main/lessons/django_blog/lesson.md)
 
+-   Python data structures like Lists, Dictionaries and their methods
+-   Python Classes and Inheritance
+-   Django [URLs](https://docs.djangoproject.com/en/3.2/topics/http/urls/) to understand how to capture parameters in your views
+-   Django [Templates](https://docs.djangoproject.com/en/3.2/ref/templates/language/)
+-   [Relational Databases]()
+-   Django [Models](https://github.com/Justice-Through-Code/fall_2021/blob/main/lessons/django_todo/lesson.md)
+-   Django [Models with many-to-many and one-to-many relationships](https://github.com/Justice-Through-Code/fall_2021/blob/main/lessons/django_blog/lesson.md)
```
Comment (lschachter) [inline projects/wow/wow.md:208]: these still reference fall

**Comment by lschachter** [comment]
oh. @ashiq-r31 do you want to make those changes or should i?

File: projects/wow/wow.md [hunk 2]
```diff
@@ -2,203 +2,207 @@
...
 ## Overview
 
-Like the Blog App and To-do List App from the class challenges, your WOW project should have CRUD functionality (the app can create, read, update, and delete data dynamically based on user interactions). 
-* This time, however, what your app does is up to you! For example:
-    * Review site for books, movies or consumer goods
-    * Recipe site for specific cuisine
-    * Curator of specific services in your area (social services, hair salons, coffee shops, etc)
-    * Online community board for jobs and volunteering
-* You'll want to think carefully about what functionality you want to give your app ahead of time, and the JTC team will help you workshop your ideas to come up with your project. 
+Like the Blog App and To-do List App from the class challenges, your WOW project should have CRUD functionality (the app can create, read, update, and delete data dynamically based on user interactions).
```
[the remark this answers is not in the document]
Comment (ashiq-r31) [reply projects/wow/wow.md:11]: resolving this since @YusufBolden updated this

File: projects/wow/wow.md (hunk 1 above)
Comment (ashiq-r31) [reply projects/wow/wow.md:208]: you could probably do this? It seems like a new lesson markdown wasn't uploaded to replace the blog one

**Pull request merged**